packages/dds/matrix/src/sparsearray2d.ts

```diff
@@ -43,7 +43,7 @@ const r0c0ToMorton2x16 = (row: number, col: number): number =>
 	(r0ToMorton16(row) | c0ToMorton16(col)) >>> 0;
 
 type RecurArrayHelper<T> = RecurArray<T> | T;
-type RecurArray<T> = RecurArrayHelper<T>[];
+export type RecurArray<T> = RecurArrayHelper<T>[];
```

Comment:
Used in `matrix.ts` - is there any reason that this shouldn't be exported?

Reply:
Should be fine IMO - we're not exporting it from the package.